streams/src/test/java/org/apache/kafka/streams/kstream/internals/KStreamPrintTest.java

```diff
@@ -56,11 +56,11 @@ public void setUp() {
     }
 
     @Test
-    public void testPrintKeyValueWithName() {
-        KeyValueMapper<Integer, String, String> mapper = new KeyValueMapper<Integer, String, String>() {
+    public void testPrintStreamWithProvidedKeyValueMapper() {
+        final KeyValueMapper<Integer, String, String> mapper = new KeyValueMapper<Integer, String, String>() {
             @Override
-            public String apply(Integer key, String value) {
-                return String.format("%d, %s", key, value);
+            public String apply(final Integer key, final String value) {
+                return String.format("(%d, %s)", key, value);
```

Comment:
I removed the first test because it was the same and one of the differences I saw (writing the email on dev list) were these parenthesis (no sense :-)) Then I deleted the first test and leaving this with parenthesis ... so I'm going to delete them ;)